profile.html

```diff
@@ -26,55 +32,74 @@
                 </div>
 
                 <div class="menuElements">
+                  <ul>
                     <li class="communityPage_title">Community</li>
-                    <li><a id="login">Logout</a></li>
-                    <li><a id="signUp">Profile</a></li>
-                    <li><a href="contact.asp">Settings</a></li>
-                    <li><a id="aboutPage">About</a></li>
+                    <li><a id="login" href="#">Logout</a></li>
+                    <li><a id="signUp" href="#" >Profile</a></li>
+                    <li><a href="settings.html" >Settings</a></li>
+                    <li><a id="aboutPage" href="#" >About</a></li>
                     <!-- have a search bar here  -->
+                  </ul>
```

Comment:
for an example of how to make the ajax calls, look at the about.html page and the ajax call made to the community page. the js code is in the index.js 

Let me know if you have any questions about this.